renku_notebooks/api/classes/user.py

```diff
@@ -63,7 +73,8 @@ def __init__(self, headers):
         )
         if not self.authenticated:
             return
-        self.gitlab_client = Gitlab(current_app.config["GITLAB_URL"], api_version=4)
+        self.git_url = os.environ["GITLAB_URL"]
```

Comment:
afaik reading environment variables only in config.py and then use the app config in the application is considered best practice in flask.